Sources/TokamakCore/Views/Containers/List.swift

```diff
@@ -120,7 +120,7 @@ public struct _ListRow {
   public static func listRow<V: View>(_ view: V, _ style: ListStyle, isLast: Bool) -> some View {
     (style as? ListStyleDeferredToRenderer)?.listRow(view) ??
       AnyView(view.padding([.trailing, .top, .bottom]))
-    if !isLast {
+    if !isLast && style as? SidebarListStyle == nil {
```

Comment:
You should probably just add a `var addDividers: Bool { get }` to the `ListStyleDeferredToRenderer` protocol, then provide a default value of `true`.

Reply:
I added "hasDividers" to the ListStyle. It wasn't clear how to add it to ListStyleDeferredToRenderer and have the code be clean. 

Here is where the property is checked:

```
@ViewBuilder
  public static func listRow<V: View>(_ view: V, _ style: ListStyle, isLast: Bool) -> some View {
    (style as? ListStyleDeferredToRenderer)?.listRow(view) ??
      AnyView(view.padding([.trailing, .top, .bottom]))
    if !isLast && style.hasDividers {
      Divider()
    }
  }
}
```